README.md

```diff
@@ -1,25 +1,54 @@
-# Pixel
+# ACM@UIUC Pixel
 
 <img src="photo.jpg" width=500>
 
-A 128x128 crowdsourced display.
+A 128x128 crowd-sourced display.
+
+# Endpoints
+
+Base API URL: [http://pixel.acm.illinois.edu](http://pixel.acm.illinois.edu).
+
+Rate Limit: 2 requests per minute.
+
+## POST /image/pixel/
+
+Displays the image contained at the URL on the ACM@UIUC Pixel display.
```

Comment:
Please keep the endpoint as just `/`.  No reason to break everybody's scripts.  Also, is the endpoint description correct?